site/content/blog/markdowndb-basics-tutorial-2023.md

```diff
@@ -0,0 +1,39 @@
+# Learn how to use MarkdownDB with our First Tutorial!
+
+![Gif](/blog/markdowndb.gif)
```

Comment:
Generally would move image slightly below lead paragraph so does not block lead.